src/controllers/resource-store/utils/route-checks/index.ts

```diff
@@ -3,7 +3,5 @@ import { Route } from '../../../../common/types';
 export const routeHasResources = (route: Route | null): boolean =>
   !!(route && route.resources && route.resources.length > 0);
 
-export const routeHasChanged = (
-  prev: Route | null,
-  next: Route | null
-): boolean => prev !== next;
+export const routeHasChanged = (prev: Route, next: Route): boolean =>
+  prev.name !== next.name || prev.group !== next.group;
```

Comment:
`name` should always be unique so checking `group` is an unnecessary step

Reply:
While I agree, given we do not enforce unique names, I wonder if we should have `name` + `path` check (just to be on the safer side). Also `group` is something we use internally but never documented. Just noticed how `Route` type also has `canTransitionIn/Out`, something we never ported

Reply:
Sounds good. I've added `name` + `path` check. Also, removed canTransitionIn/Out and group from types.
I noticed that type `JiraRoute` doesn't have `group` and `canTransitionIn`. I guess when we bump RRR in Jira, we will remember to update `JiraRoute`. Flow'll complain anyway 😀.